### ISSUE TITLE
Missing NAACCR concept for concept_code 1296@03-89

### ISSUE THREAD

**Issue by CSRessel** [OHDSI/OncologyWG#105]
Hello, I’ve been unable to find on Athena the NAACCR value corresponding to `concept_code = 1296@03-89`, in response to NAACCR variable of `concept_id = 35918370`, and believe this value may missing

**Comment by mgurley** [comment]
This issue is caused because this NAACCR variable has not been correctly identified as a 'Numeric' concept.  It should have a 'Has Type' entry in the CONCEPT_RELATIONSHIP to the 'Numeric' concept.  And there should not be entries for 'One regional lymph node examined' and 'Two regional lymph nodes examined'.  And '90 or more regional lymph nodes examined' should be handled by entries in the new CONCEPT_NUMERIC table.

**Comment by denyskaduk** [comment]
During the investigation of such objects, we found this: 560 "Order number - hospital" http://datadictionary.naaccr.org/default.aspx?c=10#560. We have some confusion regarding these values. On the one hand, it is useful to store the values as numeric, but we will skip the separation of “non-malignant tumors” and “malignant tumors” because we do not have such units. And if we create new values for each answer, it will be messy.

**Comment by dimshitc** [comment]
Just to clarify a bit:
we're going to create all possible concepts for these values

01 | First of two or more malignant primaries
02 | Second of two or more malignant primaries
03 | Third of three or more malignant primaries
10 | Tenth of ...
...
and for 60 and above the same approach but with benign neoplasm.

**Comment by cgreich** [comment]
Hm. This really is something that should go into the SPECIMEN table, and then connected to the MEASUREMENT. Should we do that?

**Comment by dimshitc** [comment]
So, SPECIMEN will have entities "tumor number 1", "tumor number 2"
then these entities should be connected to CONDITION (using fact_relationship) where we keep diagnosis and to MEASUREMENT (using measurement modifiers) where we keep stages and other cancer properties.
Looks cumbersome, isn't it?

**Comment by mgurley** [comment]
Yes, I don't think we want to put 560 "Order number - hospital" in the Specimen domain.  This is just an ordinal designation of the count of malignant or non-malignant  tumors tracked for the patient at the hospital.  And where the current tumor in question falls in that ordinal series.  It resides in a NAACCR section 'Hospital-Specific' that we have deemed out of scope and decided to place all variables in the Observation domain and make non-standard.
See here the view of NAACCR variables with sections:

http://datadictionary.naaccr.org/default.aspx?c=7

We made a decision to only concentrate on the sections 'Cancer Identification', 'Stage/Prognostic Factors', 'Treatment-1st Course' and 'Follow-up/Recurrence/Death'.

 This does not mean we should not try to get the correct list of answers for the item 560 and @dimshitc  is correct it should just be

> 
> we're going to create all possible concepts for these values
> 
> 01 | First of two or more malignant primaries
> 02 | Second of two or more malignant primaries
> 03 | Third of three or more malignant primaries
> 10 | Tenth of ...
> ...
> and for 60 and above the same approach but with benign neoplasm

**Comment by denyskaduk** [comment]
We also found item 380 'SEQUENCE NUMBER--CENTRAL' with similar values, so we do the samefor this Variable.
